cmd/server/files.go

```diff
@@ -60,16 +60,20 @@ func getCashLetterId(w http.ResponseWriter, r *http.Request) string {
 
 func getFiles(logger log.Logger, repo ICLFileRepository) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
+		if requestID := moovhttp.GetRequestID(r); requestID != "" {
+			logger.Set("requestID", requestID)
+		}
+
 		w = wrapResponseWriter(logger, w, r)
 
 		files, err := repo.getFiles() // TODO(adam): implement soft and hard limits
 		if err != nil {
-			logger.Log("files", fmt.Sprintf("error getting ICL files: %v", err), "requestID", moovhttp.GetRequestID(r))
-			moovhttp.Problem(w, err)
+			logged := logger.LogErrorf("error getting ICL files: %v", err)
+			moovhttp.Problem(w, logged.Err())
 			return
 		}
 		requestID := moovhttp.GetRequestID(r)
-		logger.Log("files", fmt.Sprintf("found %d files", len(files)), "requestID", requestID)
+		logger.Logf("found %d files requestID=%s", len(files), requestID)
```

Comment:
I don't think you need to add `requestID` here since you set it above.

Reply:
Oh you're right. I thought I caught all of those.